README.md

```diff
@@ -1 +1,39 @@
-# camp2-helpers
+# Camp2 Helpers
+
+This package modifies the [expect](https://jestjs.io/docs/en/expect) and wrap it in a custom package in order to give an extra option to add a message when jest tests fails in Typescript.
+It is based the package [jest-expect-message](https://www.npmjs.com/package/jest-expect-message) for custom use.
+
+## Usage
+
+`fwl-expect` is not yet available on npm.
+
+(the rest is commented and will be used once the package has a more final form)
```

Comment:
I think we can get rid of this message, we don't need to document what is not existing ^^